R/get_schema.R

```diff
@@ -60,7 +60,16 @@ get_schema.Id <- function(obj,  ...) {
 #' @rdname get_schema
 get_schema.PqConnection <- function(obj, temporary = FALSE,  ...) {
   if (isTRUE(temporary))  {
```

Comment:
Not within the scope of the PR (and may be my old code 😇 ), but since `temporary` defaults to `FALSE` and therefore cannot (easily) be `NULL`, is this not a nice opportunity to skip a step?

```suggestion
  if (temporary)  {
```